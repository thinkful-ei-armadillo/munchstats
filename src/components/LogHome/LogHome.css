.event{
  border: 1px black solid;
  border-radius: 5px;
  padding: 10px;
  background: linear-gradient(to right, whitesmoke 91%, #0E4749 0%);
  margin-bottom: 10px;
  box-shadow: 1px 1px 3px;
}

.trash {
  color:whitesmoke;
}

.eventTitle{
  display: flex;
  justify-content: space-between
}

.eventTime{
  margin: 5px 0 5px 0;
}

.linkContainer{
  display:flex;
  flex-direction: column;
  width:95%;
  margin:5% auto 0;
  justify-content: space-around;
  min-height: 225px;
  max-width:400px;
}

.eventsHeading{
  margin-bottom: 50px;
}