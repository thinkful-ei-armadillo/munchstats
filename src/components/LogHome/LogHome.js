import React, { Component } from 'react'
import './App.css'

export default class LogHome extends Component {

  render() {
    return (
      <div>
        
      </div>
    );
  }
}
