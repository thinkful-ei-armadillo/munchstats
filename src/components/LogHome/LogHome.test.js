import React from 'react';
import LogHome from './LogHome';
import {MemoryRouter} from 'react-router-dom';
import {mount} from 'enzyme';
import renderer from 'react-test-renderer';

describe('<LogHome />', () => {
  it('Renders without crashing', () => {
    mount(<MemoryRouter><LogHome /></MemoryRouter>);
  });
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MemoryRouter><LogHome/></MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});