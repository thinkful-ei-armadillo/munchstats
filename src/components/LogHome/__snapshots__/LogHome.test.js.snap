// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogHome /> renders the UI as expected 1`] = `
Array [
  <section
    className="goBack"
  >
    <span
      className="back_button"
      onClick={[Function]}
    >
      <i
        className="fas fa-chevron-left"
      />
    </span>
  </section>,
  <section
    className="dashboard"
  >
    <div
      className="linkContainer"
    >
      <a
        className="button"
        href="/logmeal"
        onClick={[Function]}
      >
        Log a meal that you've created
      </a>
      <a
        className="button"
        href="/logsnack"
        onClick={[Function]}
      >
        Log a single item snack
      </a>
    </div>
    <h2
      className="eventsHeading"
    >
      Today's activity:
    </h2>
  </section>,
]
`;
